src/App.css

```diff
@@ -1,4 +1,317 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&display=swap');
+
 body {
   padding: 0;
   text-align: center;
+  width: 100%;
+  margin: auto;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+#root {
+  width: 100%;
+  position: absolute;
+  top: 0;
+  background-color: #fafafa;
+}
+
+.navBar {
+  display: flex;
+  margin: auto;
+  padding: 1rem 6rem;
+  justify-content: space-between;
+  background-color: white;
+  border-bottom: solid 1px #e8e8e8;
+}
```

Comment:
- [ ] Please 🙏🏽 add te login icon to b similar to the Zeplin design,

| Zeplin | Your design |
|---|---|
| ![image](https://github.com/zamcham/Bookstore/assets/58642949/de5466f0-f8be-400a-8897-bb612cf36acb) | ![image](https://github.com/zamcham/Bookstore/assets/58642949/42ccd53f-3547-403d-9167-3f70b2396926) |